README.md

```diff
@@ -22,7 +22,39 @@ at `localhost:8080`. The incoming request may be a
 just a simple HTTP GET/POST request. The invoked user function can be
 `async` but that is not required.
 
-## Function Signatures
+## The Function Interface
```

Comment:
It should probably be documented that you can also change the health checks path using either environment variables are the `liveness.path/readiness.path` properties in the function

Reply:
hmmm,  good question.  i don't have to strong of feelings either way, but probably leaning more towards the way it currently is. function code overrides env variables

Reply:
We might also want to add `port` and `logLevel` as properties on the `Function` object so that the behavior is the same in terms of precedence. Default is superseded by configuration (in func.yaml, on the command line, or passed options to `start()`), which is superseded by environment variables.

Reply:
> hmmm, good question. i don't have to strong of feelings either way, but probably leaning more towards the way it currently is. function code overrides env variables

I think the motivation for env variables overriding function config is that the configuration would typically be set up for production environments, and coded into bash scripts or other pipeline tooling. For local development this can be overridden without changing the production settings by adjusting env vars - therefore no changes to the code.